src/javascript/app_2/App/Components/Elements/DataTable/data-table.jsx

```diff
@@ -61,7 +67,8 @@ class DataTable extends React.PureComponent {
                         TableData
                         :
                         <Scrollbars
-                            style={{ width: '100%', height: 'calc(100vh - 35px)' }}
+                            style={{ width: '100%' }}
```

Comment:
Why is it hard coded here?

Reply:
@ahkuma `Scrollbars` is a plugin, we need to pass props to it so it'll configure itself, we can't control it via CSS.